src/codeu/chat/server/Controller.java

```diff
@@ -33,6 +33,11 @@
 import codeu.chat.util.Uuid;
 
 public final class Controller implements RawController, BasicController {
+  //Please modify the value here !!!
+  public static final byte USER_TYPE_CREATOR = 0x07;
+  public static final byte USER_TYPE_OWNER = 0x03;
+  public static final byte USER_TYPE_USER = 0x01;
+  public static final byte USER_TYPE_BANNED = 0x00;
 
```

Comment:
`creatorByte = 0b111;`
`ownerByte = 0b011;`
`memberByte (member not user!) = 0b001;`
`bannedByte = 0b000;`